examples/ratatui.rs

```diff
@@ -28,11 +28,27 @@ fn main() -> io::Result<()> {
                 .filter(|c| !c.accent)
                 .map(|c| "██".fg(*c)) // fg accepts any type that implements Into<Color>
                 .collect();
+            let ansi_normals: Vec<Span> = flavor
+                .ansi_colors
+                .into_iter()
+                .filter(|c| c.code < 8)
+                .map(|c| "██".fg(*c)) // fg accepts any type that implements Into<Color>
+                .collect::<Vec<Span>>();
+            let ansi_brights: Vec<Span> = flavor
+                .ansi_colors
+                .into_iter()
+                .filter(|c| c.code >= 8)
```

Comment:
Would like to move this into a `c.bright` check, figured I could leave it for later but happy to push out a new version of the palette json if you'd like me to.

Reply:
I think I've decided against the introduction is this new variable, happy to have further discussion on it but resolving for now.